homework-css-selectors/style.css

```
@@ -0,0 +1,19 @@
+label {
+    color: skyblue;
+}
+
+.box {
+    border: 1px solid green;
+}
+
+.email {
+    background-color: blue;
+}
+
+.order {
+    background-color: green;
+}
+
+.tabel {
+    background-color: salmon;
+}
```

Este foarte buna si decizia stilizarii prin intermediul claselor

Just for your info, aici mi-ar fi placut sa vad faptul ca stii sa folosesti css attribute seletors prezentati de George

```suggestion
label {
    color: skyblue;
}

input[type="text"] {
    border: 2px solid green;
}
input[type="email"] {
    background: blue;
}
input[type="submit"] {
    background: green;
}

.tabel {
    background-color: salmon;
}
```